### PR TITLE
Bug 1196194 - Avoding switching to a new app while switching via edge gesture r=mhenretty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b6e64835a0c500d55a5402bde62e6011493a642

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5bf09fce69bedfac0526c96c6c22a32a07cd0245

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=839bab8f77d83d4975960472fe5bdc278279bb9d

File: apps/system/js/app_window_manager.js [hunk 1]
```diff
@@ -730,6 +731,7 @@
     },
 
     '_handle_sheets-gesture-begin': function() {
+      this.sheetTransitioning = true;
```
Comment (etiennesegonzac) [inline apps/system/js/app_window_manager.js]: nit: `this._sheetTransitioning` and "declare" it with a comment like the `_activeApp` private var.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fc8cc9955117ca668b31117235ed5595334a59b8

**Pull request merged**